cov.el

```diff
@@ -240,10 +238,10 @@ code's execution frequency"
   (let ((gcov (cov--coverage)))
     (if gcov
         (let* ((lines (mapcar 'cov--parse (cov--read (car gcov))))
-               (max (reduce 'max (cons 0 (mapcar 'cl-second lines)))))
+               (max (cl-reduce 'max (cons 0 (mapcar 'cl-second lines)))))
```

Comment:
Function `reduce` was not known using `ert-runner`, bus it is an alias to `cl-reduce`, which is known.